libs/ledger-live-common/src/hw/index.ts

```diff
@@ -2,10 +2,14 @@ import { EMPTY, merge } from "rxjs";
 import type { Observable } from "rxjs";
 import { catchError } from "rxjs/operators";
 import Transport from "@ledgerhq/hw-transport";
```

Comment:
i know it's annoying because right now it has been defined in 2 places, but we should start importing those types from `types-devices`
```
import { DeviceModel } from "@ledgerhq/types-devices";
```

🥸

Reply:
indeed, on it

Reply:
fixed